src/main/java/uk/gov/hmcts/reform/sendletter/entity/BasicLetterInfo.java

```diff
@@ -10,6 +10,7 @@
     private String service;
     private String status;
     private String type;
+    private String encryptionKeyFingerprint;
```

Comment:
reusing the same model as for 'stale letters'.
This field is expected by the pending letters model.